install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -349,6 +350,12 @@ data:
   #  - portmap (Enables HostPort support for Cilium)
   cni-chaining-mode: {{ .Values.global.cni.chainingMode }}
 
+{{- if hasKey .Values "enableIdentityMap"}}
+  enable-identity-map: {{ .Values.global.enableIdentityMap | quote }}
+{{- else if (ne $enableIdentityMap "true") }}
+  enable-identity-map: "false"
+{{- end }}
```

Comment:
@tgraf has the freshest understanding of exactly how to format these options, would be good to get a quick look from him whether it's actually matching our latest approach.